src/main/java/org/datadog/jenkins/plugins/datadog/DatadogBuildListener.java

```diff
@@ -129,6 +126,11 @@ public final void onStarted(final Run run, final TaskListener listener) {
     }
   }
 
+  private final boolean isJobTracked(final String jobName) {
```

Comment:
Nvm, solved in this cherry-pick PR: https://github.com/DataDog/jenkins-datadog-plugin/pull/29